src/App.js

```diff
@@ -1,7 +1,8 @@
 import MainLayout from './Components/MainLayout'
+import { BrowserRouter as Router } from 'react-router-dom'
 
 function App() {
-    return <MainLayout />
+    return <Router><MainLayout/></Router>
```

Comment:
This `Router` component is not needed since you are already using the other one with the routes inside the `MainLayou` component